src/pages/signin/SignInPage.tsx

```diff
@@ -200,6 +217,11 @@ function SignInPageInner({credentials, account, activeClients = [], preferredLoc
     let welcomeText = '';
     const headerText = translate('login.hero.header');
 
+    const userLogin = Str.removeSMSDomain(credentials?.login ?? '');
+
+    // replacing spaces with "hard spaces" to prevent breaking the number
+    const userLoginToDisplay = Str.isSMSLogin(userLogin) ? formatPhoneNumber(userLogin).replace(/ /g, '\u00A0') : userLogin;
```

Comment:
NAB since you didn't add this logic to begin with, but can we move the `.replace(/ /g, '\u00A0')` inside of `formatPhoneNumber`?